src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.commands;
+
+/**
+ * Clears the address book.
```

Comment:
Perhaps this is more like `Sorts the names in the address book in ascending/descending order` =P?